app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    if len(command_parts) != 3:
+        return
+
+    command_line, start, end = command_parts
```

Comment:
start, end
What means start, end? It is some steps of what?
Please choose better names.
source_path, destination_path